### PR TITLE
fix(name-registry): removed fee

### LINKED ISSUE #49: Name registry: fee unnecessary when adding or removing

### PR DESCRIPTION
**Pull request by ngyam**
Closes #49

### CODE REVIEW

**Comment by ngyam** [comment]
@danzipie so the registry was meant for public use by Parity originally, that's why each name reservation has a fee.
- it doesn't make sense in our case since we own the contract and we are the only ones allowed to create entries. Requiring us to pay a fee for us is just super annoying
- we didn't change this before because we didn't want to touch an already audited contract. but since I already had to modify it because of other issues, with this effort we can remove this one as well. the auditors pointed this out as well

File: contracts/registry/SimpleRegistry.sol [hunk 1]
```diff
@@ -100,7 +93,6 @@ contract SimpleRegistry is Ownable, MetadataRegistry, OwnerRegistry, ReverseRegi
         external
         payable
```
Comment (hai-ko) [inline contracts/registry/SimpleRegistry.sol]: If there is no fee payable can be removed

Comment (ngyam) [reply contracts/registry/SimpleRegistry.sol]: true dat

**Comment by ngyam** [comment]
@hai-ko done

**Pull request merged**